streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBTimestampedStore.java

```diff
@@ -92,14 +79,27 @@ void openRocksDB(final DBOptions dbOptions,
                 } catch (final RocksDBException fatal) {
```

Comment:
Not exactly related to this PR, but I don't see where we are ever creating a new column family. Is there a way to do it besides the RocksDB#createColumnFamily method, or am I just missing it somewhere?

Reply:
Nevermind, duh, it's right above 😅